### PR TITLE
RGRIDT-809: Adding calculated columns

### PR DESCRIPTION
**Pull request by glundgren93**
This PR is for adding calculated columns.

[Sample page](url)

### What was done
* Created Calculated column
* Created Calculated field features that handle formulas

### Checklist
* [x] tested locally
* [ ] documented the code
* [x] clean all warnings and errors of eslint
* [x] requires changes in OutSystems (if so, provide a module with changes)
* [x] requires new sample page in OutSystems (if so, provide a module with changes)

### CODE REVIEW

File: code/src/WijmoProvider/Features/CalculatedField.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                                    col.columnType ===
+                                        OSFramework.Enum.ColumnType.Number
+                            )
+                            // get binding
+                            .map((col) => col.config.binding)
+                            .indexOf(value) !== -1
+                );
+
+            if (!isValid) {
+                throw `The content of ${header} column is not being displayed because the given formula is not valid. Formula values must be bindings to Currency or Number columns or numeric values written as text.
```
Comment (tmbp95) [inline code/src/WijmoProvider/Features/CalculatedField.ts:67]: A good one to be converted to an error code in the future! (near future)

File: code/src/WijmoProvider/Columns/CalculatedColumn.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                columnID,
+                new OSFramework.Configuration.Column.ColumnConfigAdditional(
+                    configs,
+                    extraConfig
+                )
+            );
+            this._columnEvents =
+                new OSFramework.Event.Column.ColumnEventsManager(this);
+
+            // set custom binding
```
Comment (bmartinho) [inline code/src/WijmoProvider/Columns/CalculatedColumn.ts:21]: Could you please just comment here what is the logic to create the binding name?

**Review by bmartinho — COMMENTED** [review]
Just add the comment and it is a LGTM!

File: code/src/WijmoProvider/Features/CalculatedField.ts (hunk 1 above)
Comment (bmartinho) [reply code/src/WijmoProvider/Features/CalculatedField.ts:67]: Correct, will create a task in JIRA to implement error code logic in Data Grid Component

File: code/src/WijmoProvider/Columns/CalculatedColumn.ts (hunk 2 above)
Comment (glundgren93) [reply code/src/WijmoProvider/Columns/CalculatedColumn.ts]: done

**Review by bmartinho — APPROVED** [review]
LGTM

**Review by tmbp95 — APPROVED** [review]

**Pull request merged**